spec/openapi.yml

```diff
@@ -70,6 +70,44 @@ paths:
               schema:
                 $ref: '#/components/schemas/NamespaceList'
 
+  /events:
+    get:
+      operationId: getRawEvents
+      parameters:
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offset'
+      summary: List all received OpenLineage events.
+      description: Returns a list of OpenLineage events.
+      tags:
+        - Events
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LineageEventList'
+
+  /events/{namespace}:
+    parameters:
+      - $ref: '#/components/parameters/namespace'
+    get:
+      operationId: getRawEvents
+      parameters:
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offset'
+      summary: List all received OpenLineage events in particular namespace.
```

Comment:
can you clarify the ordering? Offset might be expensive from a query perspective